File: Project-Euler-Java/ProjectEuler005.java
import java.util.ArrayList;
import java.util.List;

public class ProjectEuler005 {
    private static final int dividerLimit = 20;
    public int run () {
        long x = 1;
        long ans;
        List<Integer> tallyCounter = new ArrayList<>();

        while(true) {
            for (int i = 1; i <= dividerLimit; i++) {
                if (x % i == 0) {
                    tallyCounter.add(i);
                } else {
                    tallyCounter.clear();
                    break;
                }
            }

            if (tallyCounter.size() == dividerLimit) {
                System.out.println("Answer: " + x);
                ans = x;
                break;
            }
            x++;
        }

        for (int i = 1; i <= dividerLimit; i++) {
            System.out.println(ans + "/" + i + "=" + ans / i);
        }
        return (int) ans;
    }
    public static void main(String[] args) {
        System.out.println(new ProjectEuler005().run());
    }
}

File: Project-Euler-Python/ProjectEuler005.py
dividerLimit = 20
x = 1
ans = int
tallyCounter = list()

while True:
    for i in range(1, dividerLimit + 1):
        if x % i == 0:
            tallyCounter.append(i)
        else:
            tallyCounter.clear()
            break

    if len(tallyCounter) == dividerLimit:
        print("Answer: ", x)
        ans = x
        break
    x += 1
